.github/workflows/ci-build.yml

```diff
@@ -6,25 +6,51 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [11]
```

Comment:
Build with Java 16 has errors. Created ticket #119 to fix them.

Reply:
Can you remove the matrix? I think the plan for java 17 was to add a separate build to all projects...